File: app/src/main/java/com/appchamp/wordchunks/ui/game/fragments/GameFinishedFrag.java
package com.appchamp.wordchunks.ui.game.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.appchamp.wordchunks.R;


public class GameFinishedFrag extends Fragment {

    public GameFinishedFrag() {
        // Requires empty public constructor
    }

    public static GameFinishedFrag newInstance() {
        return new GameFinishedFrag();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        View root = inflater.inflate(R.layout.frag_game_finished, container, false);
//        CardView cvRateUs = (CardView) root.findViewById(R.id.rlNextLevel);
//        cvRateUs.setOnClickListener(v ->
//                getContext().startActivity(
//                        new Intent(
//                                Intent.ACTION_VIEW,
//                                Uri.parse("market://details?id=" + getActivity().getPackageName())))
//        );
        return root;
    }
}

File: app/src/main/java/com/appchamp/wordchunks/ui/game/fragments/GameFinishedFrag.kt
package com.appchamp.wordchunks.ui.game.fragments

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.appchamp.wordchunks.R
import kotlinx.android.synthetic.main.frag_game_finished.*
import org.jetbrains.anko.browse


class GameFinishedFrag : Fragment() {

    companion object {
        fun newInstance() = GameFinishedFrag()
    }

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        return inflater?.inflate(R.layout.frag_game_finished, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        rlRateUs.setOnClickListener {
            context.browse("market://details?id=${activity.packageName}")
        }
    }
}
